architecture/README.md

```diff
@@ -33,127 +30,135 @@ types of roles. The TiDB community has the following organizations:
 
 ## PMC
 
-TBD
+PMC (Project Management Committee) functions as the core management team that oversees the TiDB community. The PMC has additional responsibilities over and above those of a committer. These responsibilities ensure the smooth running of the project. PMC members are expected to review code contributions, participate in strategic planning, approve changes to the governance model and manage the copyrights within the project outputs. See [Governance](../GOVERNANCE.md) for more details about PMC and [PMC List](./pmc-list.md) for a list of incumbent PMC members.
 
 ## TiDB Developer Group
 
-### Contributor
+As the cornerstone of community development, the TiDB Developer Group consists of these roles: Maintainer, Committer, Reviewer, Active Contributor, and Contributor. Each role takes corresponding responsibilities in the community. Collectively they play an important role in the robust development of TiDB.
 
-A Contributor refers to the person who contributes to the following projects:
+### Contributor
 
-* TiDB: https://github.com/pingcap/tidb
-* Parser: https://github.com/pingcap/parser
-* TiKV: https://github.com/tikv/tikv
-* TiSpark: https://github.com/pingcap/tispark
-* PD: https://github.com/pingcap/pd
-* Docs: https://github.com/pingcap/docs
-* Docs-cn: https://github.com/pingcap/docs-cn
+New contributors should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
 
-#### How to become a Contributor?
+#### How to become a Contributor
 
-If a PR (Pull Request) submitted to any one of the above projects by you is
-approved and merged, then you become a Contributor.
+- merged at least 1 PR for any project in the [PingCAP Organization](https://github.com/pingcap)
 
 You are also encouraged to participate in the projects in the following ways:
 
-* Actively answer technical questions asked by community users in github issues.
-* Help test the projects.
-* Help review the pull requests (PRs) submitted by others.
-* Help improve technical documents.
-* Submit valuable issues.
-* Report or fix known and unknown bugs.
-* Participate in the existing discussion about features in the roadmap, and
-  have interest in implementing a certain feature independently.
-* Write articles about source code analysis and usage cases for the
-  projects.
+- Actively answer technical questions asked by community users in GitHub issues.
+- Help test the projects
+- Help review the pull requests (PRs) submitted by others
+- Help improve technical documents
+- Submit valuable issues
+- Report or fix known and unknown bugs
+- Write articles about source code analysis and usage cases for the - projects.
+
+#### Responsibilities and privileges
+
+- Become a member in the SIG
```

Comment:
I'm not sure whether it's proper to be a responsibility of contributor.

Reply:
Here it's a privilege